dom-renderables/DOMElement.js

```diff
@@ -204,7 +204,7 @@ DOMElement.prototype.onDismount = function onDismount() {
  * @return {undefined} undefined
  */
 DOMElement.prototype.onShow = function onShow() {
-    this.setProperty('display', 'block');
+    if (this._node.isShown()) this.setProperty('display', 'block');
```

Comment:
This is an unnecessary change IMO. When `DOMElement#onShow` is being called, we can assume that the Node is being shown. No need to check again.